test/test_mps.py

```diff
@@ -8216,6 +8218,22 @@ def get_results(device):
         self.assertEqual(cpu_input_grad, mps_input_grad)
         self.assertEqual(cpu_weight_grad, mps_weight_grad)
 
+    # # Failures due to precision issues, enable after resolving from mps
```

Comment:
a = torch.tensor([-616], device="mps",dtype=torch.int32)
b = torch.tensor([28], device="mps",dtype=torch.int32)
z = a.floor_divide(b)
is showing -23, after install mps with fast math off, it shows the proper result (-22)